src/yang/ietf-qos-action.yang

```diff
@@ -531,7 +531,7 @@ module ietf-qos-action {
               leaf tail-drop {
                 type empty;
                 description
-                 "tail drop algorithm";
+                 "Tail drop algorithm";
```

Comment:
Since the type is empty, this should say

 "If defined, tail drop algorithm should be applied.".

But more importantly, why is this of type empty while something like ECN is a boolean?